lenskit-core/src/main/java/org/grouplens/lenskit/data/sql/JDBCRatingDAO.java

```diff
@@ -136,8 +136,14 @@ public JDBCRatingDAO(Connection dbc, SQLStatementFactory sfac, boolean close) {
         userStatement = new CachedPreparedStatement(dbc, statementFactory.prepareUsers());
         itemStatement = new CachedPreparedStatement(dbc, statementFactory.prepareItems());
         for (SortOrder order : SortOrder.values()) {
-            eventStatements.put(order, new CachedPreparedStatement(dbc,
-                                                                   statementFactory.prepareEvents(order)));
+            switch (order) {
+            case ASCENDING:
```

Comment:
OK. just remember to fix it later since making ascending and descending the same case sounds problematic.